test_bifrost.sh

```diff
@@ -3,9 +3,11 @@
 export BIFROST_DB_KEY="${BIFROST_DB_KEY:-$1}"
 export BIFROST_RUN_DIR="${BIFROST_RUN_DIR:-$PWD}"; \
 export BIFROST_CONFIG_DIR="${BIFROST_CONFIG_DIR:-$PWD}"; \
+export BIFROST_DIR=$BIFROST_RUN_DIR; \
 cd samples; \
 bash download_S1.sh; \
 cd $BIFROST_RUN_DIR; \
+mkdir $BIFROST_RUN_DIR/bifrost_test_output;
```

Comment:
In the case of run launcher it should ensure the folder is created as well, if it doesn't then there's a bug with that component